src/ray/gcs/redis_context.h

```diff
@@ -151,30 +160,29 @@ class RedisContext {
   /// \param args The vector of command args to pass to Redis.
   /// \param redis_callback The Redis callback function.
   /// \return Status.
-  void RunArgvAsync(std::vector<std::string> args,
-                    RedisCallback redis_callback = nullptr);
+  virtual void RunArgvAsync(std::vector<std::string> args,
```

Comment:
why virtual?

Reply:
This is for mocking the function in the stacked change, I'll just do it there.

Reply:
Yea, let's do it there.